nightingale_perception/config/movo_body_filter.yaml

```diff
@@ -0,0 +1,25 @@
+cloud_filter_chain:
+- name: body_filter
+  type: robot_body_filter/RobotBodyFilterPointCloud2
+  params:
+     #frames/fixed: "odom"
+     frames/sensor: "camera_link" # might be camera_depth_optical_link
+     sensor/point_by_point: False
+     sensor/min_distance: 0.05
+     body_model/inflation/scale: 1.07
+     body_model/inflation/padding: 0.03
```

Comment:
seems reasonable to me. pretty close to defaults. any reason were not using defautls? did u just use ur intuitiion?

Reply:
I thought it would be good to be slightly conservative in case the urdf is not exactly as the real robot